docs/metrics/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,106 @@
+# Getting Started with OpenTelemetry .NET in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md),
+[OpenTelemetry.Exporter.Prometheus.AspNetCore](../../../src/OpenTelemetry.Exporter.Prometheus.AspNetCore/README.md),
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md),
+and
+[OpenTelemetry.Instrumentation.AspNetCore](../../../src/OpenTelemetry.Exporter.Console/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console --prerelease
```

Comment:
```suggestion
dotnet add package OpenTelemetry.Exporter.Console
```

Reply:
This was copied from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-aspnetcore/README.md#getting-started-with-opentelemetry-net-in-5-minutes---aspnet-core-application.

Let me use a separate PR to address it (I don't know which one do we want, guess its debatable).